### PR TITLE
[SMALLFIX] Better organization of integration scripts.

### PR DESCRIPTION
**Pull request by jsimsa**
This PR moves all scripts related to integrations to `${ALLUXIO_HOME}/integration/[module]/bin`

### CODE REVIEW

**Comment by aaudiber** [comment]
I suggest grepping for the name of every renamed script to find the call sites

**Comment by jsimsa** [comment]
@aaudiber, PTAL

**Comment by jsimsa** [comment]
Jenkins, test this please

File: docs/_includes/Running-Alluxio-on-Mesos/alluxio-mesos.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ```bash
-$ ./integration/bin/alluxio-mesos.sh mesosMaster:5050 // address of Mesos master
+$ ./integration/mesos/bin/alluxio-mesos-start.sh mesosMaster:5050 // address of Mesos master
```
Comment (yupeng9) [inline docs/_includes/Running-Alluxio-on-Mesos/alluxio-mesos.md]: Why do we have such rename, and `alluxio-yarn.sh` is not changed?

Comment (jsimsa) [inline docs/_includes/Running-Alluxio-on-Mesos/alluxio-mesos.md]: the motivation for that change was that we also had a script for stopping mesos and the two scripts were called `alluxio-mesos.sh` and `stop-alluxio-framework.sh` ... now they are called `alluxio-mesos-start.sh` and `alluxio-mesos-stop.sh` .... for yarn, AFAIK, we do not have a script for stopping the application master ... if you would prefer, I would be happy to rename `alluxio-yarn.sh` to `alluxio-yarn-start.sh` and adding `alluxio-yarn-stop.sh` (I might need to ask @aaudiber or @apc999 for help with that)

Comment (yupeng9) [inline docs/_includes/Running-Alluxio-on-Mesos/alluxio-mesos.md]: This makes sense. Thanks for the explanation.

**Pull request merged**